includes/wc-product-functions.php

```diff
@@ -1048,13 +1048,17 @@ function wc_get_price_to_display( $product, $args = array() ) {
 		array(
 			'qty'   => 1,
 			'price' => $product->get_price(),
+			'price_location' => 'shop'
 		)
 	);
 
 	$price = $args['price'];
 	$qty   = $args['qty'];
+	$price_location = $args['location'];
```

Comment:
In the default args above you are using `price_location`, and here you are looking for `location`.

Reply:
done